src/configure.ac

```diff
@@ -3080,6 +3080,29 @@ else
   AC_MSG_RESULT(no)
 fi
 
+dnl ---------------------------------------------------------------------------
+dnl Using hunspell
+dnl ---------------------------------------------------------------------------
+AC_MSG_CHECKING(--with-hunspell argument)
+AC_ARG_WITH([hunspell],
+            AS_HELP_STRING([--without-hunspell],
+                           [ignore presence of hunspell and disable it]))
+AS_IF([test "x$with_hunspell" != "xno"],
+      [
+       AC_MSG_RESULT($with_hunspell)
+       if pkg-config --exists hunspell ; then
```

Comment:
I would suggest using pkg-config's autoconf macros instead (e.g., https://github.com/jamessan/vim/commit/4c89bbe0bbaa5772b23c2882c7036728db90fa3e).  That will better handle things like cross-compilation which should be invoking different binaries than the plain `pkg-config`.

Reply:
concerning pkg-config's autoconf macros http://tirania.org/blog/archive/2012/Oct-20.html and besides current ``configure.ac`` doesn't use them (even for Gnome, Gtk, etc. libraries). So, this is expressly no.

Reply:
Noted.  Thanks for the information.  #2513 should handle the part I was concerned about.